<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButtonLogin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditUser">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPsw">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;User</string>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
   <property name="buddy">
    <cstring>lineEditUser</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="passwd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Passwd</string>
   </property>
   <property name="buddy">
    <cstring>lineEditPsw</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="labelMac">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Mac</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="labelIp">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Ip</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit_2</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="labelPort">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Port</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit_3</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDay">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>60</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHexi">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>150</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPort_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Hexi</string>
   </property>
   <property name="buddy">
    <cstring>lineEditHexi</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="labelMac_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>60</y>
     <width>69</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;DayOfWeek</string>
   </property>
   <property name="buddy">
    <cstring>lineEditDay</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="labelIpYear">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Year</string>
   </property>
   <property name="buddy">
    <cstring>lineEditYear</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditYear">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>100</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditUser</tabstop>
  <tabstop>lineEditPsw</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEditDay</tabstop>
  <tabstop>lineEditYear</tabstop>
  <tabstop>lineEditHexi</tabstop>
  <tabstop>pushButtonLogin</tabstop>
  <tabstop>pushButtonExit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
